perses/app/fah_generator.py

```diff
@@ -133,7 +135,7 @@ def relax_structure(temperature,
                     positions,
                     nequil=1000,
                     n_steps_per_iteration=250,
-                    platform_name='OpenCL',
+                    platform_name='CPU',
```

Comment:
We probably shouldn't change the default platform here, since we should still prefer the GPU if possible.
This may need to be addressed more generally later by allowing OpenMM to simply use the fastest platform by default rather than specifying the platform, but this is a bit complex since we need at least `mixed` precision, which is not the default platform property.